domain-server/src/DomainServer.cpp

```diff
@@ -2784,6 +2799,9 @@ std::pair<bool, QString>  DomainServer::isAuthenticatedRequest(HTTPConnection* c
 
     QVariant adminUsersVariant = _settingsManager.valueForKeyPath(ADMIN_USERS_CONFIG_KEY);
     QVariant adminRolesVariant = _settingsManager.valueForKeyPath(ADMIN_ROLES_CONFIG_KEY);
+    QString httpPeer = connection->peerAddress().toString();
```

Comment:
`httpAddress` would be a better variable name, I think.

Reply:
I think that's a bit confusing in the context, since we're dealing with HTTP already and one can get confused about which address. This is the "peer" -- the one on the other side of the connection. Except for that, we're dealing with our own end in this part of the code.

Reply:
OK, then `httpPeerAddress` would be better, I think, to indicate that it's an IP address rather than some peer object in subsequent code.

Reply:
Done!